fdbserver/VersionedBTree.actor.cpp

```diff
@@ -3797,16 +3810,23 @@ class DWALPager final : public IPager2 {
 	ACTOR static Future<Void> clearRemapQueue_impl(DWALPager* self) {
 		// Wait for outstanding commit.
 		wait(self->commitFuture);
+		self->setOldestReadableVersion(self->getLastCommittedVersion());
+		wait(self->commit(self->getLastCommittedVersion() + 1));
```

Comment:
@sfc-gh-satherton I'm not sure why I need to bump the commit version by 1, but without it the oldest readable version does not move beyond the latest write, and the while loop below will stuck. Please advice if I'm not doing it correctly.

Reply:
It's just because the commit version must always advance for a commit, there are some things that depend on it.